flutter/lib/common.dart

```diff
@@ -1701,7 +1701,7 @@ Future<bool> restoreWindowPosition(WindowType type,
   }
   pos ??= bind.getLocalFlutterOption(k: kWindowPrefix + type.name);
 
-  var lpos = LastWindowPosition.loadFromString(pos);
+  var lpos = LastWindowPosition.loadFromString(pos!);
```

Comment:
This is not safe, because exception will be thrown if pos is null.

Reply:
so can i make chenge the data type of pos from String to String?

Reply:
My ide does not report error since `pos ??= bind.getLocalFlutterOption(k: kWindowPrefix + type.name);` making sure it is not null. Changing to `pos!` will cause error in my ide.

Reply:
I can use try catch on that